net-print/fax4cups/fax4cups-1.29-r1.ebuild

```diff
@@ -24,14 +25,13 @@ RDEPEND="${DEPEND}
 	app-admin/sudo
 "
 
-S=${WORKDIR}/fax4CUPS-${PV}
-
 src_install() {
 	doman fax4CUPS.1
 
 	exeinto $(cups-config --serverbin)/backend
 	insinto /usr/share/cups/model
 
+	local i
 	for i in hylafax efax mgetty-fax; do
 		if use $i
```

Comment:
Nit: ${i} on this and further uses, and I prefer "if ...; then" (no new line), and it's mor eocnsistent with previous usage above anyway